### PR TITLE
801 frontend add interface for admins to create new events

### PR DESCRIPTION
**Pull request by bcho892**
# Form

image

This component will need to be modified to show slight differences for editing, as it can be reused for editing an event. #779

We can start using this to test the fetching all events endpoint

### CODE REVIEW

**Review by asun555 — APPROVED** [review]
LGTM

File: client/src/services/Storage/StorageService.ts
Comment (jeffplays2005) [inline client/src/services/Storage/StorageService.ts:1]: Could possibly use imgur for storage?

**Review by jeffplays2005 — APPROVED** [review]
Just a few minor suggestions(I also hope this isn't a safari issue too), any way we could have those date boxes aligned? 

Screenshot 2024-10-09 at 10 57 09 PM

Perhaps some padding adjustments between the two headings?
Screenshot 2024-10-09 at 10 57 58 PM

Diff1:
image
Diff2:
Screenshot 2024-10-09 at 11 00 21 PM


NB: These aren't important changes!

Looks clean otherwise !!

File: client/src/services/Storage/StorageService.ts
Comment (bcho892) [reply client/src/services/Storage/StorageService.ts:1]: i would prefer a free service like that, do you think theres enough information about an API that we could make a wrapper for?

Comment (bcho892) [reply client/src/services/Storage/StorageService.ts:1]: likely should be handled in #803

**Pull request merged**